README.md

```diff
@@ -33,11 +34,8 @@ struct Repeat: ParsableCommand {
         }
     }
 }
-
-Repeat.main()
```

Comment:
The current sample code returns error. Shouldn't `Repeat.main` be added back and `@mail` be removed?
<img width="1512" alt="Screen Shot 2021-11-22 at 2 28 28 PM" src="https://user-images.githubusercontent.com/7886503/142811093-edaae370-8243-4dca-b2fe-d4604ef5f98b.png">

Reply:
Repeat main